main.go

```diff
@@ -38,13 +38,14 @@ func main() {
 	broker := brokers.NewKafkaBroker(cfg.GetBrokerInfo())
 	defer broker.CloseConnections()
 
-	sndr := push.NewHTTPSender(1)
-
-	mgr := push.NewManager(broker, store, sndr)
```

Comment:
If this is part of what remains from the all push capability then we should record in a Jira task for clean-up at a later stage.